docs/dev/parallel.md

```diff
@@ -166,10 +177,19 @@ try {
 </div>
 
 
-## System Level
+### System Level
 
 A system-wide default parallelism for all execution environments can be defined by setting the
 `parallelism.default` property in `./conf/flink-conf.yaml`. See the
 [Configuration]({{ site.baseurl }}/setup/config.html) documentation for details.
 
+## Setting the Maximum Parallelism
```

Comment:
I added an "Attention" block. @rmetzger, do you think this is now good to merge?